internal/store/model/inventory_stats.go

```diff
@@ -21,18 +21,20 @@ type OsStats struct {
 	Total int
 }
 
+type SourceBreakdown map[string]int
+
 type StorageCustomerStats struct {
 	TotalByProvider map[string]int
 	Domain          string
 }
 
 type InventoryStats struct {
-	Vms                        VmStats
-	Os                         OsStats
-	TotalCustomers             int
-	TotalAssessmentsByCustomer map[string]int
-	TotalInventories           int
-	Storage                    []StorageCustomerStats
+	Vms                                VmStats
+	Os                                 OsStats
+	TotalCustomers                     int
+	TotalAssessmentsByCustomerBySource map[string]SourceBreakdown
```

Comment:
This is highly inefficient.  O(n^2). 
I propose a simple struct like: 
```
type CustomerAssessments struct {
    AgentCount int
    RvToolCount int
}
TotalAssessmentsByCustomerBySource map[string]CustomerAssessments
```
You can use a switch after that to set the right label